data/npc/lib/npcsystem/modules.lua

```diff
@@ -1203,4 +1203,51 @@ if Modules == nil then
 		end
 		return true
 	end
+
+	VoiceModule = {
+		voices = nil,
+		voiceCount = 0,
+		lastVoice = 0,
+		timeout = nil,
+		chance = nil,
+		npcHandler = nil
```

Comment:
This is not necessary unless you want to assign it in `init` and use the handler in `callbackOnThink` instead of the npc constructor `Npc()`

Reply:
so should I remove entire 1207 to 1214? or just 1213?

Reply:
I assume 1213 only, I think it got there due to copy-and-pasting.